final release/format_data.py

```diff
@@ -0,0 +1,208 @@
+
+# coding: utf-8
+
+# In[45]:
+
+
+import numpy as np
+import pickle
+import matplotlib.pyplot as plt
+import h5py
+from skimage.transform import resize
+from astropy.io import ascii
+#import ask_first
+from sklearn.utils import shuffle
+from keras.utils import np_utils
+from astropy.coordinates import SkyCoord
+import astropy.units as u
+
+import sys, os
+
+
+# In[46]:
+
+def get_data(first_path, second_path, dim = 256, seed = 0, conf_matrix = True, flip_train_test = False):
+    # Loading data
+    data = h5py.File(first_path, 'r')
```

Comment:
Use a `with` statement to make sure this closes correctly.

Reply:
clarify, please

Reply:
`with h5py.File(first_path, 'r') as data:`

Reply:
cheers